### DISCUSSION TITLE
How to convert picture metadata into picture objects?

### DISCUSSION THREAD

**Discussion by LastMonody** [pymupdf/PyMuPDF#1455] [Looking for help] [answered]
Excuse me, when I analyze PDF, in order to extract paragraph information, I use page get_text ('block ') method, but there is "(0.0, 0.0, 261.0, 996.0, '<image: DeviceRGB, width: 268, height: 1024, bpc: 8>', 0, 1)" meta information of the picture in the returned result. I want to know how I convert this into the array format of the picture?

**Comment by JorjMcKie** [comment]
This is a "Discussions" item.

**Comment by JorjMcKie** [comment, marked as answer]
If you use the "blocks" option, only image meta information is available.
To get image content, use another option like "dict" or "rawdict".

**Reply by LastMonody** [reply]
Thank you!
